### ISSUE TITLE
Head pose normalization

### ISSUE THREAD

**Issue by rafikg** [hysts/pytorch_mpiigaze#39]
Hi @hysts,
 Why the head pose normalization is applied only  during test. I did not seeany call for  HeadPoseNormalizer anywhere in the training ?

**Comment by hysts** [comment]
Hi, @rafikg 

That's because we are using the normalized images contained in the dataset during training.

**Comment by rafikg** [comment]
@hysts Yes, indeed. Thanks

**Comment by rafikg** [comment]
@hysts I tried to play the `config.gaze_estimator.normalized_camera_distance`. The only vlaue that works is 1.0. Is the value that you used to normalize data for training?

**Comment by hysts** [comment]
@rafikg 

The parameters for MPIIFaceGaze are from [this repo](https://git.hcics.simtech.uni-stuttgart.de/public-projects/opengaze/-/blob/master/src/gaze_estimator.cpp#L89). I'm not the author of the MPIIGaze/MPIIFaceGaze paper, but the repo seems to be created by the authors, so I assume those are the values used to normalize the training data.
FYI, the parameters for MPIIGaze are written in [section 4.2 of the MPIIGaze paper](https://arxiv.org/abs/1711.09017).

See also #34
